hbase-server/src/main/java/org/apache/hadoop/hbase/master/HMaster.java

```diff
@@ -1047,6 +1077,29 @@ private void finishActiveMasterInitialization(MonitoredTask status)
     // Set master as 'initialized'.
     setInitialized(true);
 
+    if (tableFamilyDesc == null && replBarrierFamilyDesc == null) {
```

Comment:
In case one is not there also, need to create?

Reply:
I was thinking that too but for HBase 1 to 2.3+ upgrade, both will be null for sure so with this case, we are targeting this specific upgrade case.